app/assets/stylesheets/_commons.css.scss

```diff
@@ -25,14 +25,14 @@ body {
   }
 
   a { color: $gray; }
-  a.highlight { color: $new-bright-green; }
-  a:hover { color: $new-bright-green; }
+  a.highlight { color: $bright-green; }
+  a:hover { color: $bright-green; }
 
-  #meal, #man, #measure { fill: $new-blue; }
+  #meal, #man, #measure { fill: $blue; }
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line